moped-editor/src/views/staff/EditStaffView.js

```diff
@@ -44,6 +44,7 @@ const EditStaffView = () => {
     error: userQueryError,
   } = useQuery(GET_USER, {
     variables: { userId },
+    fetchPolicy: "cache-and-network",
```

Comment:
Thank you for unraveling [the correct fetch policy](https://www.apollographql.com/docs/react/data/queries/#supported-fetch-policies). It's far from obvious to me which is right. 🙏🙏